airbrakes/mock/mock_imu.py

```diff
@@ -26,19 +27,27 @@ def __init__(self, log_file_path: Path, real_time_simulation: bool):
         :param real_time_simulation: Whether to simulate a real flight by sleeping for a set period, or run at full
             speed, e.g. for using it in the CI.
         """
-        super().__init__(log_file_path, real_time_simulation)
+        # We don't call the parent constructor as the IMU class has different parameters, so we manually start the
+        # process that fetches data from the log file
 
-        # Limit how big the queue gets when doing an integration test, because we read the file
-        # much faster than update(), sometimes resulting thousands of data packets in the queue,
-        # which will obviously mess up data processing calculations. We limit it to 15 packets, which
+        # If it's not a real time sim, we limit how big the queue gets when doing an integration test,
+        # because we read the file much faster than update(), sometimes resulting thousands of data packets in
+        # the queue, which will obviously mess up data processing calculations. We limit it to 15 packets, which
         # is more realistic for a real flight.
-        if not real_time_simulation:
-            self._data_queue: multiprocessing.Queue[IMUDataPacket] = multiprocessing.Queue(15)
+        self._data_queue: multiprocessing.Queue[IMUDataPacket] = multiprocessing.Queue(
+            MAX_QUEUE_SIZE if real_time_simulation else NOT_REAL_TIME_MAX_QUEUE_SIZE
+        )
```

Comment:
wait wdym, shouldn't this work too? II just made it a ternary operator cause its cleaner

Reply:
it works yes, but I don't like the variable name at all. The previous simple `if real_time_simulation: ...` was easier to understand at first glance

Reply:
I'll admit the var name is ass, but because it's not using IMU's constructor, we should specify the max queue sizes. What about `MOCK_MAX_QUEUE_SIZE` or `SIMULATION_MAX_QUEUE_SIZE`

Reply:
SIMULATION_MAX_QUEUE_SIZE